radixdlt-java/src/test/java/com/radixdlt/client/util/HashTest.java

```diff
@@ -0,0 +1,26 @@
+package com.radixdlt.client.util;
+
+import com.radixdlt.client.core.util.Hash;
+import okio.ByteString;
+import org.junit.Test;
+
+import java.io.UnsupportedEncodingException;
+
+import static junit.framework.TestCase.assertEquals;
+
+public class HashTest {
+
+    @Test
+    public void testDoubleSha256Hash() {
```

Comment:
Generally I think it's a good idea to name your unit test cases on the form:

```java
@Test
public void if_i_do_this__then_that_should_happen() {
    // Test something
}
```

I think this helps both you, as a test writer, to stay on focus, but even more so me, as a reviewer or peer developer, to understand what you intend to test and to verify that you're really testing what you're claiming.

This is a matter of taste though, and to be honest I'd value consistency (with other tests in the suite) over clarity in this case.